sys/netinet/ip_carp.c

```diff
@@ -819,12 +825,17 @@ carp_send_ad_locked(struct carp_softc *sc)
 #ifdef INET
 	if (sc->sc_naddrs) {
 		struct ip *ip;
-
+		uint8_t ip_tos = IPTOS_DSCP_CS7;
+		
+		
 		m = m_gethdr(M_NOWAIT, MT_DATA);
 		if (m == NULL) {
 			CARPSTATS_INC(carps_onomem);
 			goto resched;
 		}
+                if (!V_carp_dscp) {
+			ip_tos = IPTOS_LOWDELAY;
+		}
```

Comment:
The curved brackets are not needed here.  And "else" here IMHO could be slightly more readable then assigning the other value above.